authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -297,31 +476,41 @@ func prepareCheckAccessRequestBody(req *authzv1.SubjectAccessReviewSpec, cluster
 			}
 		}
 	*/
-	checkaccessreq := CheckAccessRequest{}
+
 	var userOid string
 	if oid, ok := req.Extra["oid"]; ok {
 		val := oid.String()
 		userOid = val[1 : len(val)-1]
+		if !isValidUUID(userOid) {
+			return nil, errors.New("oid info sent from authentication module is not valid")
+		}
 	} else {
 		return nil, errors.New("oid info not sent from authentication module")
 	}
+	groups := getValidSecurityGroups(req.Groups)
+	username = req.User
+	var actions []oputil.AuthorizationActionInfo
+	actions, err := getDataActions(req, clusterType, dataActionsMap)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error while creating list of dataactions for check access call")
+	}
+	var checkAccessReqs []*CheckAccessRequest
+	for i := 0; i < len(actions); i += 200 {
+		j := i + ActionBatchCount
+		if j > len(actions) {
+			j = len(actions)
+		}
 
-	if isValidUUID(userOid) {
+		checkaccessreq := CheckAccessRequest{}
+		checkaccessreq.Subject.Attributes.Groups = groups
 		checkaccessreq.Subject.Attributes.ObjectId = userOid
-	} else {
-		return nil, errors.New("oid info sent from authentication module is not valid")
+		checkaccessreq.Actions = actions[i:j]
+		klog.Infof("checkaccessreq.Actions : %v %d", checkaccessreq.Actions, len(checkaccessreq.Actions))
```

Comment:
verbose level?

Reply:
missed removing this. Was added for testing. removed